.github/workflows/compilation.yml

```diff
@@ -200,9 +205,6 @@ jobs:
         automatic_release_tag: "latest"
         title: "${{ steps.version.outputs.version }}"
         files: |
-          OPNPS2LD-${{ steps.version.outputs.version }}.7z
-          OPNPS2LD-VARIANTS-${{ steps.version.outputs.version }}.7z
-          OPNPS2LD-LANGS-${{ steps.version.outputs.version }}.7z
           OPNPS2LD.7z
```

Comment:
@israpps you didnt add debug artifact into pre-release

Reply:
> @israpps you didnt add debug artifact into pre-release

you want it on pre release too?

Ok then.